drivers/SmartThings/philips-hue/src/handlers/attribute_emitters.lua

```diff
@@ -22,22 +22,9 @@ local AttributeEmitters = {}
 local device_type_emitter_map = {}
 
 ---@param light_device HueChildDevice
----@param light_repr HueLightInfo|HueZigbeeInfo
+---@param light_repr HueLightInfo
 local function _emit_light_events_inner(light_device, light_repr)
   if light_device ~= nil then
-    if light_repr.status then
```

Comment:
Was this code moved?

Reply:
Yeah it was moved in to the `connectivity_check` function.